screens/device_integration.py

```diff
@@ -0,0 +1,27 @@
+import streamlit as st
+
+def app():
+    st.title("Device Integration")
+
+    st.write("""
+    Integrate your sleep data with the following devices for enhanced tracking and personalized recommendations. 
+    Below are some of the popular devices we support:
+    """)
+
+    col1, col2, col3 = st.columns(3)
+
+    with col1:
+        st.image("images/apple_watch.png", caption="Apple Watch")
```

Comment:
for the images, ideally they should be more uniform in size, you can't specify height in streamlit but maybe 'use_column_width=True' might improve it